plugin/trino-clickhouse/src/main/java/io/trino/plugin/clickhouse/ClickHouseClient.java

```diff
@@ -99,6 +102,7 @@
         extends BaseJdbcClient
 {
     private final boolean mapStringAsVarchar;
+    private static final Logger log = Logger.get(ClickHouseClient.class);
```

Comment:
nit: the project prefers the order `static final`, `static`, `final` then others. So this should be moved to the top.